crates/orchestrator/src/config.rs

```diff
@@ -710,20 +730,34 @@ impl<KEY: SignatureKey, E: ElectionConfig> From<ValidatorConfigFile> for HotShot
 
 impl<KEY: SignatureKey> Default for HotShotConfigFile<KEY> {
     fn default() -> Self {
+        let staked_committee_nodes: usize = 5;
```

Comment:
Is 5 significant here? Or is it just a decided on number? If it's a constant, should it perhaps be notated as such?